File: monitoring/src/main/java/com/example/monitoring/user/adapter/port/in/request/CreateUserRequest.java
package com.example.monitoring.user.adapter.port.in.request;

import com.example.monitoring.user.application.port.in.request.CreateUserRequestDto;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CreateUserRequest {

    private String name;
    private int age;

    public CreateUserRequestDto toRequestDto() {
        return CreateUserRequestDto.builder()
                .name(name)
                .age(age)
                .build();
    }
}

File: kotlin-spring-boot/src/main/kotlin/com/hello/kotlin/user/presentation/request/CreateUserRequest.kt
package com.hello.kotlin.user.presentation.request

import com.hello.kotlin.user.application.request.CreateUserRequestDto

data class CreateUserRequest(
    val name: String,
    val age: Int,
    val address: String,
) {

    fun toRequestDto() = CreateUserRequestDto(
        name,
        age,
        address
    )
}